samples/errorhandling/Core_7/PlatformLauncher/PlatformLauncher.Core.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Why netcoreapp if the platform launcher requires full framework?

Reply:
Explained to Daniel on Slack that package is multi-targeted but just informs it can't do anything via logging on non-Windows platforms for now.